src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -13,35 +18,38 @@ public class FlywheelSubsystem extends SubsystemBase {
      * 1 Follower TalonFX (Right)
      * Total: 2 TalonFX motors
      */
+    TalonFX left = new TalonFX(FlywheelConstants.LEFT_CAN_ID);
+    TalonFX right = new TalonFX(FlywheelConstants.RIGHT_CAN_ID);
 
     public FlywheelSubsystem() {
+        right.follow(left);
     }
 
     /*
      * Stop the flywheel from moving
      */
     public void stopFlywheel() {
+        left.set(ControlMode.PercentOutput,0);
```

Comment:
You should be using `.setNeutralOutput` instead of setting percent to 0 because the flywheel will try to PID to zero which is unnecessary. `.setNeutralOutput` just stops giving power to the motor so it slowly comes to a stop.